test/src/test/java/org/corfudb/runtime/object/transactions/AbstractTransactionContextTest.java

```diff
@@ -93,16 +93,13 @@ void setupCounters() {
     @Test
     public void ensureEmptyWriteSetIsNotWritten() {
         TXBegin();
-        long result = getRuntime().getObjectsView().TXEnd();
+        getRuntime().getObjectsView().TXEnd();
         ILogData ld =
                 getRuntime()
                         .getAddressSpaceView()
                         .peek(0);
         assertThat(ld)
                 .isNull();
-
-        assertThat(result)
-                .isEqualTo(Address.NON_ADDRESS);
```

Comment:
I tried to. Turns out this particular test isn't run by itself it is a test template that runs a bunch of different transaction types - Optimistic, WriteAfterWrite, Snapshot and its variants and in a few of these because of the order in which the tests run, the address returned ends up being a valid one causing unpredictable test failures. 
It might be better to run a separate test to capture this scenario rather than add it into this place which is used in several different tests.